### ISSUE TITLE
FF7Save ps4 read support 

### ISSUE THREAD

**Issue by sithlord48** [sithlord48/ff7tk#5]
FF7Save needs to be able open ps4 save correctly for reading.

**Comment by wmxp** [comment]
Yeah, the PS4 format isn't getting reverse engineered anytime soon. SW is done by [Data Power Development Limited](https://lmgtfy.com/?q=data+power+development+limited), which was responsible for the PS3 Game Genie, and done by a team out of Hong Kong. You can see their trademark registars on several databases.

Very deliberately, SW does all it's magic on online servers - uploading and downloading your saves to a remote location outside the purview of IDA and the like. This has had the added benefit of allowing them to win the cat and mouse game against Sony for over two years now since the tool was released around July 2017. Sony have changed the USB export signing method and keys several times since then, and amazingly they keep cracking it over and over. (knock on wood)

Interestingly, they had a business contract with a company in Japan for exclusive distribution there, along with exclusive use JP region saves - SW wouldn't touch them, you needed:
https://www.cybergadget.co.jp/products/4544859028212/ 

...but then some law out of your nightmares was passed over there in 2019, making save editing illegal.(I'm not even kidding!): https://kotaku.com/game-modding-illegal-in-japan-punishable-by-prison-and-1831525943 (the article names the Cyber save editor by name)

I've since noticed JP games popping up on the "newly added" SW news feed...
